enhancements/installer/assisted-installer.md

```diff
@@ -0,0 +1,349 @@
+---
+title: assisted-installer
+authors:
+  - "@avishayt"
+  - "@hardys"
+  - "@dhellmann"
+reviewers:
+  - "@beekhof"
+  - "@deads2k"
+  - "@hexfusion"
+  - "@mhrivnak"
+approvers:
+  - "@crawford"
+  - "@abhinavdahiya"
+  - "@eparis"
+creation-date: 2020-06-09
+last-updated: 2020-06-10
+status: implementable
+see-also:
+  - "/enhancements/baremetal/minimise-baremetal-footprint.md"
+---
+
+# Assisted Installer
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions
```

Comment:
I like the list of open questions, and I think it would make sense to focus the dicussion here on getting to a relatively complete list rather than trying to the open questions before this becomes a PR against openshift/enhancements. Otherwise the super useful discussion here will probably end up being repeated

i.e. is this the right list of open questions? have we captured the main ideas for each of those questions?

Reply:
Moved to line 224 based on template change.